website/templates/base.mako

```diff
@@ -80,16 +80,16 @@
 <div id="footerSlideIn">
     <div class="container">
         <div class="row">
-            <div class='col-sm-2 hidden-xs'>
```

Comment:
Why was `hidden-xs` removed? Removing it makes the logo appear incorrect on smaller screens.

![osf___capuchin_monkeys__cebus_apella__show_no_evidence_for_inequity_aversion_in_a_costly_choice_task_and_bower_json____projects_cos-projects_osf__-_vim_and__2___sara_-_flowdock](https://cloud.githubusercontent.com/assets/2379650/7189828/9f617c04-e44f-11e4-8123-552ca6be8974.png)